src/policies/accept-language.js

```diff
@@ -0,0 +1,40 @@
+import parser from 'accept-language-parser';
+import addVary from '../util/vary';
+
+const createPolicy = (options) => {
+    options = {
+        loose: true,
+        ...options,
+    };
+
+    return {
+        match(locales, { req, res }) {
+            const acceptLanguage = typeof window !== 'undefined' ? navigator.languages.join(',') : req?.headers['accept-language'];
```

Comment:
Doing that in a util will render dead-code elimination useless as webpack will no longer inline those expression to `false` or `true`. To solve this one could use [prepack](https://prepack.io/) but it would slow down the build process.

Reply:
In regards to `navigator.languages`: it seems unnecessary to do those checks as our target browsers seems to support them: https://caniuse.com/#search=navigator%20languages. I can still add it though in case someone else use this library outside our target browsers. What do you think it's best?

Reply:
It has good enough browser support to just ignore the check 👍